Packs/Malware/Scripts/CreateHashIndicatorWrapper/CreateHashIndicatorWrapper.py

```diff
@@ -0,0 +1,125 @@
+from typing import Tuple
+
+from CommonServerPython import *
+
+import traceback
+
+''' STANDALONE FUNCTION '''
+
+INCIDENT_ID = demisto.incident().get('id')
+
+XDR_ACTIONS = {
+    'allow': 'xdr-whitelist-files',
+    'block': 'xdr-blacklist-files'
+}
+
+MSDE_ACTIONS = {
+    'allow': 'Allowed',
+    'block': 'AlertAndBlock',
+}
+
+CROWDSTRIKE_ACTIONS = {
+    'allow': {'action': 'allow',
+              'description': f'Whitelisted based on XSOAR inc {INCIDENT_ID}',
+              'severity': 'low'},
+    'block': {'action': 'prevent',
+              'description': f'Blacklisted based on XSOAR inc {INCIDENT_ID}',
+              'severity': 'high'}
+}
+
+
+def create_commands(hashes: List[str], action: str) -> List[CommandRunner.Command]:
+    """
+    Create a list of `Command` of the allow/block hash command to `Cortex XDR`, `CrowdstrikeFalcon`,
+     `Microsoft Defender Advanced Threat Protection`
+
+    :param hashes: The hashes list
+    :param action: The action to have (one of {'allow', 'block'})
+    :return: A list of `Command`
+    """
+
+    msde_action = MSDE_ACTIONS.get(action)
+    crowdstrike_commands, crowdstrike_args_lst = get_crowdstrike_commands_args(hashes, action)
+    return [CommandRunner.Command(commands=XDR_ACTIONS.get(action),
+                                  args_lst={'hash_list': ','.join(hashes)}),
+            CommandRunner.Command(commands=['microsoft-atp-sc-indicator-create' for _ in hashes],
```

Comment:
This is exactly what I don't like about the `CommandRunner.Command`. We shouldn't have to make such a list if the command is the same for all args. I feel it'll be much better to offer an override, and accept `str` for `commands`, and given an str, it'll use it for all the commands.